src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let newState = { ...state };
+  const res = [];
```

Comment:
res/result/arr/array/obj/object/el/elem and etc are bad names, name should describe a variable